src/lua/init.c

```diff
@@ -28,6 +28,7 @@
  * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  */
+#include "stdbool.h"
```

Comment:
Do you need this library? I believe `bool` variables existed here before this patch.